tests/common/plugins/custom_skipif/tests_to_be_skipped_conditionally.yaml.example

```diff
@@ -0,0 +1,122 @@
+# It is example - of file with skip test cases by issue or by platform.
+# Original skip test cases file should be named "tests_to_be_skipped_conditionally.yaml"
+
+## Ignore all tests in folder platform_tests/cli if any issue(GitHub or Jira or Redmine) active or platform msn4600
+#platform_tests/cli:
+#  GitHub:
+#    - https://github.com/Azure/sonic-buildimage/issues/7602
+#    - https://github.com/Azure/sonic-buildimage/issues/7643
+#  Jira:
+#    - http://jira.aaa.com/1234
+#  Redmine:
+#    - http://redmine.bbb.com/1234
+#  Platform:
+#    platforms:
+#      - msn4600
+#
+## Ignore all tests in file platform_tests/cli/test_show_platform.py if any issue(GitHub or Jira or Redmine) active or platform msn4600
+#platform_tests/cli/test_show_platform.py:
+#  GitHub:
+#    - https://github.com/Azure/sonic-buildimage/issues/7602
+#    - https://github.com/Azure/sonic-buildimage/issues/7643
+#  Jira:
+#    - http://jira.aaa.com/1234
+#  Redmine:
+#    - http://redmine.bbb.com/1234
+#  Platform:
+#    platforms:
+#      - msn4600
+#
+## Ignore specific test platform_tests/cli/test_show_platform.py::test_show_platform_fan if any issue(GitHub or Jira or Redmine) active or platform msn4600
+#platform_tests/cli/test_show_platform.py::test_show_platform_fan:
+#  GitHub:
+#    - https://github.com/Azure/sonic-buildimage/issues/7602
+#    - https://github.com/Azure/sonic-buildimage/issues/7643
+#  Jira:
+#    - http://jira.aaa.com/1234
+#  Redmine:
+#    - http://redmine.bbb.com/1234
+#  Platform:
+#    platforms:
+#      - msn4600
+#
+## Ignore all tests in folder platform_tests/cli if any issue(GitHub or Jira or Redmine) active and platform msn4600
+#platform_tests/cli:
+#  GitHub:
+#    - https://github.com/Azure/sonic-buildimage/issues/7602
+#    - https://github.com/Azure/sonic-buildimage/issues/7643
+#  Jira:
+#    - http://jira.aaa.com/1234
+#  Redmine:
+#    - http://redmine.bbb.com/1234
+#  Platform:
+#    operand: "and"
```

Comment:
Here is a (completely optional) suggestion for what to do here.

Have the top-level header take a list instead of a dict where each item of this list is "AND-ed" together and the items within the list are "OR-ed" together (like you have it now by default)

For example to define a skip when an issue matches a Github issue AND a platform.. (each type has individual list entry)

```
platform_tests/cli:
  - GitHub:
      - https://github.com/Azure/sonic-buildimage/issues/7602
  - Platform:
      - msn4600
```

But to define a skip when an issue matches a Github issue OR a platform... (single list entry)

```
platform_tests/cli:
  - GitHub:
      - https://github.com/Azure/sonic-buildimage/issues/7602
    Platform:
      - msn4600
```

What this will allow is for any logical combination of requirements to be used to skip a test (i.e. maybe you want to only skip if both the redmine AND the github issue is open. Also if we add a sonic version checker as well we could check for github issues that are only applicable to a single sonic version.

This also allows us to go without the platform-specific "operand" logic you have. Ideally this system is extensible in that it treats each implementation of a CustomSkipIf equally.